docs/source/API.rst

```diff
@@ -164,7 +164,7 @@ Vector field (vf)
 
    vf.SparseVFC
    vf.base_vectorfield
-   vf.svc_vectorfield
+   vf.SvcVectorfield
```

Comment:
any particular reason on changing it to SvcVectorfield?

Reply:
Based on  google code style convention and other style guide, class name should be camel case and capitalized